sys/include/ztimer64.h

```diff
@@ -0,0 +1,528 @@
+/*
+ * Copyright (C) 2021 Inria
+ *               2021 Freie Universität Berlin
+ *               2021 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_ztimer
```

Comment:
Doxygen grouping seems to be off, should this be:

```suggestion
 * @ingroup     sys_ztimer
 * @defgroup     sys_ztimer64
```